exercises/go-2-b/.meta/start_2_example.go

```diff
@@ -0,0 +1,99 @@
+package go2b
+
+const maxScore = 21
+
+type Card int
+
+const (
+	Ace   Card = 11
+	King       = 10
+	Queen      = 10
+	Jack       = 10
+	Ten        = 10
+	Nine       = 9
+	Eight      = 8
+	Seven      = 7
+	Six        = 6
+	Five       = 5
+	Four       = 4
+	Three      = 3
+	Two        = 2
+)
```

Comment:
Just a little trick that could be used here:

```go
const (
	Two Card = iota + 2
	Three
	Four
	Five
	Six
	Seven
	Eight
	Nine
	Ten
	Ace

	King  = 10
	Queen = 10
	Jack  = 10
)
```

No need to change this though if you'd rather have it the straight forward way.

Reply:
Also note, that in your code above, only `Ace` is of type `Card`. The others are of type `int`.

Reply:
Thanks!